### PR TITLE
MRG: Add epochs metadata summary to HTML representation

### PR DESCRIPTION
**Pull request by hoechenberger**
Adds a new row to the `Epochs` HTML repr table, stating whether metadata is present; and if yes, how many rows and columns.

### CODE REVIEW

File: mne/html_templates/repr/_acquisition.html.jinja [hunk 1]
```diff
@@ -8,8 +8,8 @@
 {% set collapsed_row_class = "" %}
 {% endif %}
 
-<tr class="repr-section-header {{ section_class_name }}" {% if collapsed %} title="Show section" {% else %} title="Hide section" {% endif %}
-    onclick="toggleVisibility('{{ section_class_name }}')">
+<tr class="repr-section-header {{ section_class_name }}" {% if collapsed %} title="Show section" {% else %}
+    title="Hide section" {% endif %} onclick="toggleVisibility('{{ section_class_name }}')">
```
Comment (hoechenberger) [inline mne/html_templates/repr/_acquisition.html.jinja:12]: Autoformatter did that … not related to the new feature

File: tutorials/epochs/30_epochs_metadata.py [hunk 2]
```diff
@@ -88,7 +88,6 @@
 # groups.
 
 epochs.metadata["NumberOfLetters"] = epochs.metadata["NumberOfLetters"].map(int)
-
```
Comment (hoechenberger) [inline tutorials/epochs/30_epochs_metadata.py:91]: touched it for rendering

Comment (larsoner) [reply tutorials/epochs/30_epochs_metadata.py:91]: Oddly enough looking at that example I don't think `epochs` is ever just printed, but I confirmed locally that it works
Screenshot 2024-06-29 at 3 27 26 PM

**Comment by larsoner** [comment]
Thanks @hoechenberger !

**Pull request merged**